### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by ninakuni**
Overview on updates:

1) Updating 
- models
- Fitting / EM inferences
- GW-EM sampling (fix typos) 

2) New pages / topics
- injection & observational data 
- GW inference (injection & real event)
- Multi-messenger inference

### CODE REVIEW

**Review by mcoughlin — APPROVED** [review]
Looks great. A few small comments, otherwise good to go!

File: doc/GW-EM-resampling.md [hunk 1]
```diff
@@ -10,10 +10,10 @@ For estimating the source properties, the following input files are required:
 * `EMprior` - is the prior file that was used for the EM inference 
 * `EOS` - number of equation of state files which will be used in the resampling
 * `EOSpath` - path to the folder of all EOS files
-* `GWsamples` - some fiducial randomly generated posterior samples for masses $m_{1,2}$ , chirp mass $\mathcal{M}_c$, mass ratio $q$, luminosity distance $D_L$, and EOS samples,
```
[the remark this answers is not in the document]
Comment (ninakuni) [reply doc/GW-EM-resampling.md:13]: It just annoyed me that the mathematical symbols were not properly displayed, so removed those

[the remark this answers is not in the document]
Comment (mcoughlin) [reply doc/GW-EM-resampling.md:13]: Ah ok.

File: doc/data_inj_obs.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+Observatories provide observational data as a measured flux or as spectral flux density which needs to be converted to the AB magnitude system in order to be 
+used within NMMA. It is important to distinguish between different spectral filters in order to run a consistent analysis. Current available filters can be
+checked in [utils.py](https://github.com/nuclear-multimessenger-astronomy/nmma/blob/baf85f600a6e7369608c7bd6e76b9f1f4ec3d1cd/nmma/em/utils.py#L48). 
+A specific flux or respective magnitude measurement comes along with a specific time stamp which needs to be in the ISOT time format. 
+For further details, see [here](https://docs.astropy.org/en/stable/api/astropy.time.TimeISOT.html).
+
+Example: AT2017gfo data excerpt
+
+    2017-08-18T00:00:00.000 g 17.41000 0.02000
```
[the remark this answers is not in the document]
Comment (ninakuni) [reply doc/data_inj_obs.md:54]: If someone can recompute the data for some sncosmo filters, I can include those as well

[the remark this answers is not in the document]
Comment (mcoughlin) [reply doc/data_inj_obs.md:54]: @ninakuni Sent to your email. Please confirm the file seems ok to you and can commit it to replace https://github.com/nuclear-multimessenger-astronomy/nmma/blob/main/example_files/lightcurves/GW170817.dat and update this example

[the remark this answers is not in the document]
Comment (mcoughlin) [reply doc/data_inj_obs.md]: This is great, thanks!

**Pull request merged**